pkgs/applications/video/mpv/default.nix

```diff
@@ -100,37 +99,31 @@ in stdenv.mkDerivation rec {
   NIX_LDFLAGS = lib.optionalString x11Support "-lX11 -lXext "
     + lib.optionalString stdenv.isDarwin "-framework CoreFoundation";
```

Comment:
We don't need the LDFLAGS for darwin here anymore, we should check whether we can drop this for Linux aswell.